tehom/tests/test_downloads.py

```diff
@@ -0,0 +1,10 @@
+from tehom import _downloads, _persistence
+
+
+def test_download_ais_to_temp(declare_stateful):
```

Comment:
Decorate this with `@pytest.mark("slow")`

Reply:
build failed:

tehom/tests/test_downloads.py:6: in <module>
    @pytest.mark("slow")
E   TypeError: 'MarkGenerator' object is not callable

It was for this reason that I omitted the decorator on my push. The docs say something about defining the 'slow' marker in a pytest.ini file: https://docs.pytest.org/en/6.2.x/mark.html

Reply:
Weird, [it is defined as a marker](https://github.com/Jacob-Stevens-Haas/tehom/blob/960538e607e8134992425e15bd12ed4cbe087355/pyproject.toml#L37).  Let me take a look at this.

Reply:
Yeah, if you run `pytest --markers`, you can see that it should be written:
```python
@pytest.mark.slow
def test_...
```
not
```python
@pytest.mark("slow")
def test_...
```
My mistake.